python-package/lightgbm/plotting.py

```diff
@@ -304,24 +301,22 @@ def add(root, parent=None, decision=None):
         if parent is not None:
             graph.edge(parent, name, decision)
 
-    graph = Digraph(name=name, comment=comment, filename=filename, directory=directory,
-                    format=format, engine=engine, encoding=encoding, graph_attr=graph_attr,
-                    node_attr=node_attr, edge_attr=edge_attr, body=body, strict=strict)
+    graph = Digraph(**kwargs)
     add(tree_info['tree_structure'])
 
     return graph
 
 
 def create_tree_digraph(booster, tree_index=0, show_info=None, precision=None,
-                        name=None, comment=None, filename=None, directory=None,
-                        format=None, engine=None, encoding=None, graph_attr=None,
-                        node_attr=None, edge_attr=None, body=None, strict=False):
+                        old_name=None, old_comment=None, old_filename=None, old_directory=None,
```

Comment:
why do we need to change args name?
(eg. `name` -> `old_name`)

Reply:
Because it's impossible to save old behavior with positional arguments and new one with `**kwards` due to duplicate argument name. For example, `name` as positional argument and `name` as `kwarg`.

Reply:
Also, in such case we cannot check was it kwarg and should we show the warning.

Reply:
Ah, I see.
You leave these arguments for users who use positional args 👍

Reply:
Yeah! Not sure, how popular these functions are and whether this backward compatibility will be useful for someone or not. But why not, if we can? 💪